src/main/java/frc/robot/RobotContainer.java

```diff
@@ -67,24 +79,28 @@ public class RobotContainer {
 
     // Commands
     private final SendableChooser<Command> autonChooser;
-
+    private final Command balanceCommand;
     /**
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer() {
-        swerveSubsystem = new SwerveSubsystem2020();
-        // swerveSubsystem = new MissileShellSwerveSubsystem();
+        dt = new Tank(); // initialize DT of choice
+        ahrs = new AHRS(SPI.Port.kMXP);
```

Comment:
i don't think we should be constructing multiple ahrs, i don't know what the behavior is but I doubt it's good. Maybe we can have the AHRS in the Drivetrain class, and offer methods to get data from the ahrs? so the balancecommand doesn't need the ahrs either